flowdock-text.js

```diff
@@ -963,25 +963,29 @@ if (typeof FlowdockText === "undefined" || FlowdockText === null) {
     };
   };
 
-  FlowdockText.mentionsAllTags = ["@everyone", "@everybody", "@all", "@anyone", "@anybody"];
-
-  FlowdockText.mentionsAll = function(check){
+  FlowdockText.mentionsTags = function(check, tags){
     if(isArray(check)){
-      return FlowdockText.mentionsAllTags.some(function(tag) {
-        return downCase(check).indexOf(tag) !== -1;
-      });
+      if (isArray(tags)){
+        return tags.some(function(tag) {
+          return downCase(check).indexOf(tag) !== -1;
+        });
+      } else {
+        return downCase(check).indexOf(tags) !== -1;
+      }
     } else {
-      return FlowdockText.mentionsAll(FlowdockText.extractMentions(check));
+      return FlowdockText.mentionsTags(FlowdockText.extractMentions(check), tags);
     }
   };
 
+  // DEPRECATED. Will be removed. Use FlowdockText.mentionsTags instead.
```

Comment:
I think these deprecation warnings are a bit redundant since we probably anyways want to retain the special "has a team mention" or "has an everyone mention" in the future too.